cypress/e2e/CMS/$SiteSections/index.js

```diff
@@ -122,4 +122,12 @@ And('{string} appears in position {int} in the side menu tree', (label, position
 
 And('user clicks on title with the url {string} from the list of content', (contentHref) => {
     cy.get(`a[href='${contentHref}']`).click();
+});
+
+Then('the current left navigation label has url {string}', (currentHref) => {
+    cy.get('ul.usa-sidenav__sublist li').find(`a[href='${currentHref}']`).should('be.visible');
```

Comment:
this does not verify its a current page
try this instead cy.get('ul.usa-sidenav__sublist li a[aria-current="page"]').should('have.href',currentHref)

Reply:
Getting the following error: 

![image](https://user-images.githubusercontent.com/28455973/216724056-4ea0a4fc-7e55-48a9-bbf3-55a900c11d99.png)